pkg/rpc/client/rpc.go

```diff
@@ -1142,3 +1146,50 @@ func (c *Client) GetNativeContractHash(name string) (util.Uint160, error) {
 	c.cacheLock.Unlock()
 	return cs.Hash, nil
 }
+
+// TraverseIterator returns a set of iterator values (maxItemsCount at max) for
+// the specified iterator and session. If result contains no elements, then either
+// Iterator has no elements or session was expired and terminated by the server.
+// If maxItemsCount is a negative number, then the full set of iterator values
+// will be returned using several `traverseiterator` calls if needed.
```

Comment:
`traverseAll` is not a good idea in general, it can be a problem for client too (receiving too many elements). It should have a simpler model, one `TraverseIterator` call -> one set of items.

Reply:
Fixed in a separate (last) commit. But IMO, if the user doesn't want to receive too many elements, he just should set `maxItemsCount` to lower value.

Reply:
Iterator allows you to process items one by one irrespective of the number of them. 1M of items is perfectly fine to handle with iterator, but might be a problem if you try fitting all of them into memory.